recipes/spec2nii/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "spec2nii" %}
+{% set version = "0.2.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2bcf36adfe8ccc40f565ebceaa179b157f0faeb1953308763f7fd3ebdda45959
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points: 
+    - spec2nii = spec2nii.spec2nii:main
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - numpy
+    - nibabel
+    - pydicom
+    - pymapvbvd
+    - scipy
+
+test:
+  imports:
+    - spec2nii
+
```

Comment:
Since you have an entrypoint, you should also check for it in the test section, e.g.:
```
commands:
  - spec2nii --help
```

Reply:
Thanks. Will do.